### DISCUSSION TITLE
Join TWIG

### DISCUSSION THREAD

**Discussion by daudix** [GradienceTeam/Gradience#102] [General]
We need anouncment for the TWIG, and also cool screenshots

**Comment by 0xMRTT** [comment]
Maybe a screenshot like the cover on the readme

**Reply by daudix** [reply]
maybe

**Comment by 0xMRTT** [comment]
The announcement on twig need to be simple. Something like an introduction about the goal of AdwCustomizer and a bullet points list with features and one screenshot per tab 

A picture with a background and the rendering would be great.

**Comment by daudix** [comment]
not realted, but while I maked screenshots, I noticed that about window still have MIT licence

**Reply by 0xMRTT** [reply]
Thanks 🙏👍

**Reply by 0xMRTT** [reply]
Corrected

**Comment by daudix** [comment]
And funny bug

![image](https://user-images.githubusercontent.com/77155297/184175538-e04ca037-176f-4ed5-9943-b3f26a330714.png)

**Reply by 0xMRTT** [reply]
I know, it's because of my last commit. The goal is to have the ability to select if this theme will apply as dark theme or as light theme.

**Reply by 0xMRTT** [reply]
![image](https://user-images.githubusercontent.com/105598867/184186017-9281d67f-f4c3-43da-98f1-5a4f58ad6ba6.png)

**Comment by daudix** [comment]
## Somthing like this:

This week, Adwaita Manager was significantly redesigned, Thanks to @0xMRTT.

![AdwCustomizer - Colors](https://user-images.githubusercontent.com/77155297/184178512-704f8cf7-2df8-475b-8377-07d7920feb81.png)
![AdwCustomizer - Monet](https://user-images.githubusercontent.com/77155297/184178515-90920989-702c-4830-a753-f30371765329.png)
![AdwCustomizer - Advanced](https://user-images.githubusercontent.com/77155297/184178504-cc7cc71d-6644-4f0c-903c-ddd95a8b187f.png)
![AdwCustomizer - About](https://user-images.githubusercontent.com/77155297/184178706-6ffadb1f-b679-482d-a12b-25494237ea1c.png)

Adwaita Manager allows you to customize Libadwaita applications and the adw-gtk3 theme with simple UI, as well with Material You Palette generation from wallpaper.

![AdwCustomizer - Colors (Pretty Purple)](https://user-images.githubusercontent.com/77155297/184178752-b273b1f1-021b-4819-b658-343ae7c88aad.png)
![AdwCustomizer - About (Pretty Purple)](https://user-images.githubusercontent.com/77155297/184178746-51f3b350-89e4-4e14-a5be-456544a64b14.png)

Welcome screen and Plugins support will come very soon.

**Reply by daudix** [reply]
[📎 AdwCustomizer TWIG.zip](https://github.com/AdwCustomizerTeam/AdwCustomizer/files/9310107/AdwCustomizer.TWIG.zip)

**Comment by 0xMRTT** [comment]
I think that the monet generator should be in use with a file.

(Select a file, wait and take a screenshot after)

**Reply by daudix** [reply]
It don't work for some reason, app just freezes

**Reply by daudix** [reply]
Will try to build manually

**Reply by 0xMRTT** [reply]
> It don't work for some reason, app just freezes

I got this issue because the generation is slow and my computer was out of memory

**Reply by daudix** [reply]
Haha, that means that reason is PotatoBook 2011 😄

**Comment by 0xMRTT** [comment]
The generation of a theme is slow. I've added this in the label at the top but I think that we need a generate button at the same place of the drop-down in CSS. And when you click on this button, an Adw status page open with a progress bar and log output.

**Reply by daudix** [reply]
That would be nice touch!

**Comment by 0xMRTT** [comment]
# 0.1.0

![image](https://user-images.githubusercontent.com/105598867/184189322-ffbf44f2-87a6-4ea1-affb-aa3f04a39cbf.png)

**Comment by daudix** [comment]
## Some changes

This week, Adwaita Manager was significantly redesigned Thanks @0xMRTT for making it real and @daudix-UFO for providing mockups.

[Screenshots here]

Adwaita Manager allows you to customize Libadwaita applications and the adw-gtk3 theme with simple UI, as well with Material You Palette generation from wallpaper.

[Even more]

Adwaita Manager is actively developed, so there may be some bugs, feel free to report them!

Welcome screen and Plugins support will come very soon.

**Reply by 0xMRTT** [reply]
It's just perfect 👍😎🥳

**Reply by 0xMRTT** [reply]
I think that we also need to credit @ArtyIF because he created the project

**Reply by daudix** [reply]
Yes, maybe write in context of team expanding?

**Comment by daudix** [comment]
## Another edit

This week, Adwaita Manager was significantly redesigned, Thanks @0xMRTT for making it real and @daudix-UFO for providing mockups.

Adwaita Manager allows you to customize Libadwaita applications and the adw-gtk3 theme with simple UI, as well with Material You Palette generation from wallpaper.

Also this week @ArtyIF migrated Adwaita Manager to Adwaita Manager Team, which allowed to add new maintainers, @0xMRTT and @daudix-UFO

More amazing changes will come in near future!

**Comment by daudix** [comment]
## One more (Final if everything OK)

Adwaita Manager was significantly redesigned, Thanks for hard work of @0xMRTT who maked it real and @daudix-UFO for providing design mockups.

Adwaita Manager is a tool that allows you to customize Libadwaita applications and the adw-gtk3 theme with simple UI, as well with Material You Palette generation from wallpaper, Thanks @avanishsubbiah for porting Monet engine to Python!

Also, @ArtyIF has migrated Adwaita Manager to Adwaita Manager Team, which allowed to add new maintainers, @0xMRTT and @daudix-UFO

More amazing changes will come in near future!

**Reply by 0xMRTT** [reply]
It's not @allaeddineomc but @avanishsubbiah

**Reply by 0xMRTT** [reply]
Wrote Changelog in release

**Reply by daudix** [reply]
Oh my dumb grey cells.

FIXED

**Reply by 0xMRTT** [reply]
I think that it's ready to be published on TWIG 😎🥳🥳🥳🥳🎉🎉🎉

**Reply by 0xMRTT** [reply]
Do you want to publish the message on TWIG channel?

**Reply by daudix** [reply]
Hopefully I will not make any stupid typo )

**Reply by daudix** [reply]
Ok, letsss go!

**Reply by 0xMRTT** [reply]
Are you publishing ?

**Reply by daudix** [reply]
I think i'm dumb, but how to add screenshots, as a link?

**Reply by 0xMRTT** [reply]
No, just mention twig bot and upload screenshot with + button

**Reply by daudix** [reply]
So... you or me?

**Reply by daudix** [reply]
I think it done!

**Comment by 0xMRTT** [comment]
🎉 Published in TWIG thanks to @daudix-UFO 🎉

**Comment by ArtyIF** [comment]
What's TWIG? lol

**Reply by 0xMRTT** [reply]
THhis week in GNOME

**Reply by 0xMRTT** [reply]
https://thisweek.gnome.org

**Comment by daudix** [comment]
https://thisweek.gnome.org/posts/2022/08/twig-56/
